tests/testsuite/bad_config.rs

```diff
@@ -442,7 +447,13 @@ Caused by:
   could not parse input as TOML
 
 Caused by:
-  expected a table key, found a newline at line 8 column 27
+  TOML parse error at line 8, column 27
+    |
+  8 |                 native = {
+    |                           ^
+  Unexpected `
+  `
+  Expected `key`
```

Comment:
"key" is a description of what's expected, not literal text, so I think this should not have backquotes.

Also, shouldn't it be "Expected key or `}`"?

Reply:
@epage For this one, ignoring the formatting of "`key`", shouldn't the parser be expecting a possible closing brace here?

Reply:
This is a bit tricky.  If trailing commas were allowed in inline tables, we could just say to always expect a `}` where a key is expected (though trailing commas lead to their own error-reporting problems).  I wonder if `combine` can be modified to report that with their `between` combinator.

The fun of parser combinators compared to hand-rolled parser :)

Reply:
https://github.com/ordian/toml_edit/issues/296